README.md

```diff
@@ -1,4 +1,74 @@
 # OPA Spring Security Library
 
+OPA Spring Security is a library that enables usage of OPA as authorization tool in Spring applications.
```

Comment:
Suggestion: "OPA Spring Security is a library that enables using OPA for authorization in Spring applications."